docs/notebooks/figanos_multiplots.ipynb

```diff
@@ -0,0 +1,213 @@
+{
```

Comment:
<p>si j'ajoute , 'col_wrap':2 ici, les plots se dézooment.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/figanos/pull/136/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
en fait, c'est un bug xarray quand tu passes col_wrap et une projection. ex:
`ds_space['tx_max_p50'].plot(**{"x": "lon", "y": "lat", "col": "time",'col_wrap':2
                              },
                            subplot_kws={"projection": ccrs.LambertConformal()},
                            transform = ccrs.PlateCarree()
                           )`

Reply:
J'ai ouvert cet issue; leur proposition pour régler le bug est de mettre un set_extend pour l'instant https://github.com/pydata/xarray/issues/8562#issuecomment-1865189766

Reply:
I think we can add a section in the notebook documenting the bug and pointing to the issue. Thanks for asking them!